Frontend/ui-customer/src/Components/QuoteListModel/View.purs

```diff
@@ -234,6 +235,7 @@ findingRidesView state push =
       [ id (getNewIDWithTag "lottieLoaderAnim")
       , afterRender (\action-> do
                     void $ pure $ startLottieProcess lottieAnimationConfig{ rawJson = (getAssetsBaseUrl FunctionCall) <> "lottie/finding_rides_loader_with_text.json", lottieId = (getNewIDWithTag "lottieLoaderAnim") }
+                    -- void $ pure $ startLottieProcess lottieAnimationConfig{ rawJson = if (getMerchant FunctionCall == YATRI && state.vehicleVariant == "AUTO_RICKSHAW") then (getAssetsBaseUrl FunctionCall) <> "lottie/finding_rides_loader_with_text_auto.json" else (getAssetsBaseUrl FunctionCall) <> "lottie/finding_rides_loader_with_text.json", lottieId = (getNewIDWithTag "lottieLoaderAnim") }
```

Comment:
@Anuragini-2506 Please remove the commented code and if it is required then add  comments for the same.

Reply:
done